torchci/clickhouse_queries/recent_pr_workflows_query/query.sql

```diff
@@ -81,3 +89,5 @@ WHERE
   w.id in (select id from materialized_views.workflow_run_by_head_sha where head_sha in (select sha from recent_prs))
 ORDER BY
   time DESC
+-- Non experimental analyzer has problems with final...
```

Comment:
Just a thought, the need to add final into all the different places seems easy to miss.  Maybe we should adopt what @kit1980 has been doing in https://github.com/pytorch/test-infra/blob/main/torchci/clickhouse_queries/queued_jobs_by_label/query.sql#L48 by setting final to true for the whole query?

Reply:
Maybe when we get a linter working in CI

Reply:
Yeah, a linter for this would be good